sim/common/sod/items_sets/phase_3.go

```diff
@@ -162,10 +206,35 @@ var ItemSetShunnedDevoteesChainmail = core.NewItemSet(core.ItemSet{
 	Name: "Shunned Devotee's Chainmail",
 	Bonuses: map[int32]core.ApplyEffect{
 		2: func(agent core.Agent) {
-			// c := agent.GetCharacter()
+			c := agent.GetCharacter()
+			c.AddStat(stats.MeleeCrit, 1)
+			c.AddStat(stats.SpellCrit, 1)
 		},
 		3: func(agent core.Agent) {
-			// c := agent.GetCharacter()
+			c := agent.GetCharacter()
+
+			// Holy Spell Crit
+			c.OnSpellRegistered(func(spell *core.Spell) {
+				if spell.SpellSchool.Matches(core.SpellSchoolHoly) {
+					spell.BonusCritRating += 3
+				}
+			})
+
+			// Nature Bonus Proc
+			procAura := c.NewTemporaryStatsAura("The Furious Storm Proc", core.ActionID{SpellID: 449934}, stats.Stats{stats.NaturePower: 60, stats.HealingPower: 60}, time.Second*10)
```

Comment:
The healing component is only on Nature spells so I wasn't sure if we had any way to handle that properly. I know we don't really do healing sims in SoD but just wanted to call it out just in case there's another approach we can take

Reply:
We dont really care as all druid healing spells are nature afaik

Reply:
This is the Paladin/Shaman tier set 😅

Reply:
hmm, then it might be an issue if we ever add paladin healing 😅

Reply:
Would it be possible to do the healing component through `OnSpellRegistered` and checking the spell school and proc mask? Idk if there's any way to add a flat stat bonus to a spell 🤔

Reply:
I'll fix this in a future PR i have some ideas

Reply:
Nice looking forward to it!